pywhat/what.py

```diff
@@ -26,6 +26,13 @@ def main(text_input):
 
     Your text must either be in quotation marks, or use the POSIX standard of "--" to mean "anything after -- is textual input".
 
+
+    pyWhat can also search files or even the whole directory with recursion:
+
+        * what secret.txt
+
+        * what "this/is/a/path"

```

Comment:
We need single quotations here on both of them, not double `"` or no quotations. Google "Hashcat can't crack hash with `"`" to find out why Bash hates double quotation marks :-)